mache/spack/__init__.py

```diff
@@ -302,8 +295,7 @@ def get_modules_env_vars_and_mpi_compilers(machine, compiler, mpi, shell,
         Whether to include the same hdf5, netcdf-c, netcdf-fortran and pnetcdf
         as used in E3SM
 
-    include_system_hdf5_netcdf : bool, optional
-        Whether to include the system hdf5, netcdf-c, netcdf-fortran and
+        Whether to include the netcdf-c, netcdf-fortran and
         pnetcdf (not necessarily the same as E3SM)
```

Comment:
As do these 2.

Reply:
Oh, sorry, I thought I was just supposed to remove the references to `hdf5`. I see the issue now, that was my mistake, I didn't look at the whole context. Will fix that now.